### ISSUE TITLE
No softmax in Pointer class

### ISSUE THREAD

**Issue by zhangchen010295** [BangLiu/QANet-PyTorch#6]
Hello, I would like to know why do not you add the softmax operation in the last line of Pointer class:
        Y1 = mask_logits(self.w1(X1).squeeze(), mask)
        Y2 = mask_logits(self.w2(X2).squeeze(), mask)
        return Y1, Y2

Thank you!

**Comment by BangLiu** [comment]
@zhangchen010295 The loss function will add soft max.
